index.html

```diff
@@ -1,71 +1,103 @@
-    </header>
+      <section class="space-images">
+        <img src="img/blast.png" class="blast" alt="Rocket blasting off" />
+        <img src="img/sky.png" class="sky" alt="Space sky image" />
+        <img src="img/launch.png" class="launch" alt="Launching rocket" />
+      </section>
 
-    <section class="space-images">
-        <img src="img/blast.png" class="blast" alt="Rocket blasting off">
-        <img src="img/sky.png" class="sky" alt="Space sky image">
-        <img src="img/launch.png" class="launch" alt="Launching rocket">
-    </section>
-   
-    <section class="visit-planets">
-     
-      <h3>Can We Visit Planets?</h3>
-    
-      <p>Never in all their history have men been able truly to conceive of the world as one: a single sphere, a globe, having the qualities of a globe, a round earth in which all the directions eventually meet.</p>
-    
-      <p>Which there is no center because every point, or none, is center — an equal earth which all men occupy as equals. The airman's earth, if free men make it.</p>
-    </section>
-    
-    <section class="walk">
-      <div class="walk-text">
-        <h3>Let's Go For A Walk</h3>
-
-        <p>The Earth was small, light blue, and so touchingly alone, our home that must be defended like a holy relic. The Earth was absolutely round.</p>
-      
-        <p>Suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth.</p>
-      
-        <div class="btn">
-          Sign Up
-        </div>
-      </div>
-      
-      <div class="walk-img">
-          <img src="img/space-walk.png" alt="Space walking astronaut">
-      </div>
+      <section class="visit-planets">
+        <h3>Can We Visit Planets?</h3>
 
-    </section>
+        <p>
+          Never in all their history have men been able truly to conceive of the
+          world as one: a single sphere, a globe, having the qualities of a
+          globe, a round earth in which all the directions eventually meet.
+        </p>
+
+        <p>
+          Which there is no center because every point, or none, is center — an
+          equal earth which all men occupy as equals. The airman's earth, if
+          free men make it.
+        </p>
+      </section>
 
-    <footer>
-      <p>Copyright SpaceWalkers 2018</p>
-    </footer>
+      <section class="walk">
+        <div class="walk-text">
+          <h3>Let's Go For A Walk</h3>
+
+          <p>
+            The Earth was small, light blue, and so touchingly alone, our home
+            that must be defended like a holy relic. The Earth was absolutely
+            round.
+          </p>
+
+          <p>
+            Suddenly struck me that that tiny pea, pretty and blue, was the
+            Earth. I put up my thumb and shut one eye, and my thumb blotted out
+            the planet Earth.
+          </p>
+
+          <div class="btn">
+            Sign Up
+          </div>
```

Comment:
to get semantic, maybe put this element in a button tag as opposed to a div tag.